pkg/apis/extensions/types.go

```diff
@@ -35,6 +35,15 @@ import (
 	"k8s.io/kubernetes/pkg/util/intstr"
 )
 
+// The following identify resource constants for Kubernetes object types
+const (
+	// ReplicaSets, number
+	ResourceReplicaSets api.ResourceName = "replicasets"
```

Comment:
in related note, `ValidateResourceQuantityValue` also can validate if a resource is an integral value...  it would be good if we update `validation.go` to ensure it knows these are integrate values as well.